scripts/models.py

```diff
@@ -29,7 +29,7 @@ def regression():
     else:
         data_path = Path('../data')
 
-    predictor = LinearRegression(data_path)
+    predictor = LinearRegression(data_path, experiment='one_month_forecast')
```

Comment:
None of these are in the `__init__` methods in the actual models, so this script doesn't work

Reply:
(This is also causing the tests to fail: `TypeError: __init__() got an unexpected keyword argument 'experiment'`)

Reply:
this is fixed